model/lifecycle.go

```diff
@@ -1809,7 +1809,7 @@ func addNewTasks(ctx context.Context, batchTimeInfo batchTimeTasksAndVariants, v
 		}
 	}
 	if batchTimeInfo.hasAnyBatchTimeTasks() {
-		grip.Error(message.WrapError(v.UpdateBuildVariants(), message.Fields{
+		grip.Error(message.WrapError(v.UpdateBuildVariantsAndActivation(), message.Fields{
```

Comment:
Actually you can't do this here; "hasAnyBatchTimeTasks" means there are tasks that need to be activated eventually, but aren't activated now. However for these functions, all the tasks that are added are automatically activated, so you can just separately do an update for version activation outside of an if statement